python/nistoar/midas/dbio/wsgi/project.py

```diff
@@ -677,47 +678,55 @@ def __init__(self, service: ProjectService, svcapp: ServiceApp, wsgienv: dict, s
             # programming error
             raise ValueError("Missing ProjectRecord id")
 
-        
+
     def do_OPTIONS(self, path):
         return self.send_options(["GET", "POST", "PUT", "PATCH", "DELETE"])
 
     def do_GET(self, path, ashead=False):
+        path = path.strip('/')
+        if not path:
+            # "GET /<id>/acls" with no sub-path => return entire dictionary
+            try:
+                prec = self.svc.get_record(self._id)
+            except dbio.NotAuthorized:
+                return self.send_unauthorized()
+            except dbio.ObjectNotFound:
+                return self.send_error_resp(404, "ID not found",
+                                            "Record with requested identifier not found",
+                                            self._id, ashead=ashead)
```

Comment:
This try block is repeated below--that is, regardless of `path`, this code needs to be executed.  Let's move this code up above the `if not path:` and remove the same block below.  Yes?